src/api/status/package.json

```diff
@@ -18,10 +20,17 @@
     "express": "^4.17.1"
   },
   "engines": {
-    "node": ">=12.0.0"
+    "node": ">=12.0.0 <=14"
```

Comment:
Yeah, we can't use <16 with other parts of how our stuff works.

Reply:
Actually, one thing we could do is split the Dockerfile into two stages, and have the SCSS compile step use node 14 or whatever, then copy the built artifacts out to the next stage and use node 16.  That won't solve the local development case, though, if that matters?

Reply:
That should do it, something like this:

```yml
FROM node:14.18-alpine as base

WORKDIR /app

COPY . .

RUN npm install --only=production --no-package-lock

RUN npm run compile:scss

FROM node:lts-alpine as deployment

WORKDIR /app

# https://snyk.io/blog/10-best-practices-to-containerize-nodejs-web-applications-with-docker/
RUN apk add dumb-init

COPY --chown=node --from=base /app /app
USER node

CMD ["dumb-init", "node", "src/server.js"]
```

Reply:
But I'm not sure if it's a good idea to install dependencies and build using one version of node and then deploy with a different version?
I think we should start having `.nvmrc` files for scenarios like this one.

Reply:
@manekenpix So should I go with the split or your first suggestion?

Reply:
I'm not sure, Having a `.nvmrc` file would help with local development, but I'm not sure if we should do that in this PR. As for the `Dockerfile`, I'd use only one version of `node` in the container, but maybe @humphd has a better approach.

Reply:
We still have another option that is using `sass` instead of `node-sass`. It only needs a small migration work, which is replacing deprecated API. Last time I checked there were 20 lines needed replacing, not much.

Reply:
If you can do this by using standard node packages on 16+, I'd do that vs. complex Docker or dev setups.